### ISSUE TITLE
iOS webpage crash on huge fallback list (1000+ songs)

### ISSUE THREAD

**Issue by FreTimmerman** [Festify/app#226]
i'm currently testing Festify, and i have added a huge spotify playlist as fallback (1127 songs)
this works okay on Desktop (Chrome) and Android browsers. But on iOS this gives issues:
everytime you or someone else votes on a song, the page crashes.

smaller playlists work fine on my iOS device

**Comment by leolabs** [comment]
Or we need to virtualize the queue better. In the meantime, maybe we could limit the visible fallback tracks to 20 or 50 because you don't need to see the whole fallback playlist anyway :p

**Comment by NeoLegends** [comment]
Yeah, I don't want to implement pages in the classical way either. Keep the infinite-scrollable list, but load the tracks as needed.

**Comment by FreTimmerman** [comment]
to be fair, i now realise 1000+ songs is kinda overkill anyway. that's over 70 hours of continuous music, not to mention extra songs other people will add.
about 400 songs is 24+ hours, so that's plenty enough for most applications, i'd think? (i guess you can go 10 seconds without music in order to select another playlist even if it ends before your event does :p )

so maybe for now add a warning that huge playlists can have detrimental effects on user experience? until this issue is fixed properly...

**Comment by NeoLegends** [comment]
Yeah 1000 tracks is a lot, but since the data layer of Festify can handle that many tracks easily, the UI should be able to do the same. :)
